BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/FileDiagnosticsTelemetryModule.cs

```diff
@@ -103,7 +99,15 @@ public string LogFilePath
         }
 
         /// <summary>
-        /// Initializes the telemetry module.
+        /// Gets or sets the self-diagnostics configuration string used to setup this module.
+        /// </summary>
+        /// <remarks>
+        /// This module can be setup by the Self-Diagnostics Environment Variable and we want to include that string in the file.
+        /// </remarks>
+        internal string SelfDiagnosticsConfig { get; set; }
```

Comment:
This class already has public method for LogFilePath, Level etc. which would be duplicated in the SelfDiagnosticsConfig right? Can we instead simply populate the existing fields only?

Reply:
populating this field for logging purposes.
Reiley suggested to include the full string in the file header. If a user forgets to turn it off, this will indicate that it's still enabled.